third_party/geogram/cmake/FindGeogram.cmake

```diff
@@ -53,18 +53,18 @@ find_library (GEOGRAM_GFX_LIBRARY
 # system search path since it may be already installed
 # in the system
 find_library (GEOGRAM_GLFW3_LIBRARY
-                NAMES glfw3 glfw geogram_glfw3
+                NAMES glfw3 glfw geogram_glfw3 glfw3dll glfwdll
```

Comment:
A cleaner way may be to create our own FindGeogram instead of modifying the geogram's one

Reply:
I hope BL will modify its code. This is a temporary fix. If not, we will look at another solution, like your's for example.

Reply:
Does he know about changes to do? Should I send this file to him?